src/encoded/static/components/quality_metric.js

```diff
@@ -65,6 +65,34 @@ const qcAttachmentProperties = {
         { nuclear_preseq: 'Total read metric file' },
         { nuclear_preseq_targets: 'Sequencing depths file' },
     ],
+    ScAtacAlignmentQualityMetric: [
+        { mito_stats: 'Mitochondrial read statistics' },
+        { samstats: 'SAMstats QC' },
+    ],
+    ScAtacReadQualityMetric: [
+        { barcode_matching_stats: 'Barcode matching QC' },
+        { adapter_trimming_stats: 'Adapter trimming QC' },
+        { barcode_revcomp_stats: 'Barcode reverse complement detection QC' },
+    ],
+    ScAtacMultipletQualityMetric: [
+        { excluded_barcodes: 'Excluded barcodes from multiplet filtering' },
```

Comment:
I think this line needs to be removed, there is no `excluded_barcodes` property.